### DISCUSSION TITLE
Error reopening in Batocera

### DISCUSSION THREAD

**Discussion by manurafal** [AntiMicroX/antimicrox#755] [General] [answered]
I am using the application in Batocera, but when I minimize the application, to be able to run a game, I can no longer open it, since there is no system bar in batocera and I don't know how to access the application to close it. Is there a way from the command line to run it with a profile, and once I load and finish the game, I can close the application or how can I reopen the application gui once it is in the system bar?

**Comment by pktiuk** [comment, marked as answer]
Just use `killall antimicrox` to close it.    
You may also once again open AntiMicroX to reopen hidden window.  
But I think that AutoProfile may be also useful in your case.
